ipfs/src/route.rs

```diff
@@ -119,7 +119,8 @@ mod tests {
 	use ethcore::client::TestBlockChainClient;
 
 	fn get_mocked_handler() -> IpfsHandler {
-		IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
+		//IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
```

Comment:
We have git, we don't need to comment code (I assume this was an accident). Is this not compiling anymore? If it's unimplemented can we remove the function entirely?